v2/kotlin-sdk/app/src/main/java/com/mongodb/app/TemplateApp.kt

```diff
@@ -0,0 +1,25 @@
+package com.mongodb.app
+
+import android.app.Application
+import android.util.Log
+import io.realm.kotlin.mongodb.App
+
+lateinit var realmApp: App
+
+// global Kotlin extension that resolves to the short version
+// of the name of the current class. Used for labelling logs.
+inline fun <reified T> T.TAG(): String = T::class.java.simpleName
+
+/*
+*  Sets up the Realm App and enables Realm-specific logging in debug mode.
+*/
+class TemplateApp: Application() {
+
+    override fun onCreate() {
+        super.onCreate()
+//        realmApp = App.create(getString(R.string.realm_app_id))
+        realmApp = App.create("application-0-qgjpm") // TODO change to template app
```

Comment:
Why not set your R.string.realm_app_id to application-0-qgjpm?

Reply:
I prefer changing it here and keep the actual app id safe in the strings, just in case.